src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+@import '../blocks/stars';
+@import '../utils/variables';
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: block;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
```

Comment:
[Code Style Rules:](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css) Don't fix container size (if there is no such a requirement)